### PR TITLE
hook-template for tram-one hooks (and functions)

### PR DESCRIPTION
**Pull request by JRJurman**
## Summary

This PR enables another template that should make creating new custom hooks (or even basic functions) easier by providing a default `index.js`, `test.js` and source file in a named folder.

### CODE REVIEW

File: templates/tram-hook/__name__/__name__.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {{ name }} from './{{ name }}'
+
+describe('{{ name }}', () => {
+	it('should return true by default', () => {
+		expect({{ name }}()).toBe(true)
```
Comment (evajurman) [inline templates/tram-hook/__name__/__name__.test.js:5]: What the heckin' is this syntax

**Review by ethanjurman — APPROVED** [review]
I'm only a tad confused how this PR is supposed to work.

File: templates/tram-hook/__name__/__name__.test.js (hunk 1 above)
Comment (JRJurman) [reply templates/tram-hook/__name__/__name__.test.js:5]: it's the template syntax, in reality it generates a file that looks like this:
```javascript
import useTest from './useTest'

describe('useTest', () => {
	it('should return true by default', () => {
		expect(useTest()).toBe(true)
	})
})
```

Comment (evajurman) [reply templates/tram-hook/__name__/__name__.test.js:5]: Ahh!! Now it makes sense

**Pull request merged**